src/modules/eventFeedback/api.ts

```diff
@@ -0,0 +1,24 @@
+import { GetQuestionGroupsQuery } from "@/apollo/types";
+import { useQuery } from "@vue/apollo-composable";
+import gql from "graphql-tag";
+
+export const useEvents = () =>
+  useQuery<GetQuestionGroupsQuery>(gql`
+    query getQuestionGroups {
+      event(id: 1) {
```

Comment:
should be param not number 1
https://v4.apollo.vuejs.org/guide-composable/query.html#variables

Reply:
Will you add logic for this page so that I could leave this for you